tasks/extract_filepaths_for_granule/extract_filepaths_for_granule.py

```diff
@@ -4,12 +4,14 @@
 Description:  Extracts the keys (filepaths) for a granule's files from a Cumulus Message.
 """
 
+import json
 import re
 from typing import Dict, List, Union
 
+import fastjsonschema as fastjsonschema
 from aws_lambda_powertools import Logger
 from aws_lambda_powertools.utilities.typing import LambdaContext
-from run_cumulus_task import run_cumulus_task
+from fastjsonschema import JsonSchemaException
```

Comment:
combine line 11 and 14. `from fastjsonschema import fastjsonschema, JsonSchemaException` so that common imports are grouped together.

Reply:
`from fastjsonschema import fastjsonschema, JsonSchemaException`
 gives
`ImportError: cannot import name 'fastjsonschema' from 'fastjsonschema'`

Updated to 

```
import fastjsonschema as fastjsonschema
from fastjsonschema import JsonSchemaException
```